code/modules/admin/admin_investigate.dm

```diff
@@ -34,7 +34,7 @@
 
 	selected = replacetext(selected, " (empty)", "")
 
-	if(selected == "notes, memos, watchlist" && check_rights(R_ADMIN))
+	if(selected == "notes, memos, watchlist" && (check_rights(R_ADMIN)|| check_rights(R_DEV)))//This is slightly risky. Skyrat Edit
```

Comment:
Not a fan of this

Reply:
Nor am I, which is why there is a comment about it - it'll be changed to R_PC (since Useroth has plans to allow player committee to have their own tag)